src/pages/messages/TasksPage.tsx

```diff
@@ -89,6 +89,16 @@ const TasksPage = () => {
     meta: { errorMessage: t('feedback.error.get_person') },
   });
 
+  const [organizationFilter, setOrganizationFilter] = useState(
+    institutionUserQuery.data?.viewingScope?.includedUnits ?? []
+  );
+
+  useEffect(() => {
+    if (institutionUserQuery.data?.viewingScope?.includedUnits) {
+      setOrganizationFilter(institutionUserQuery.data.viewingScope.includedUnits);
+    }
+  }, [institutionUserQuery.data?.viewingScope?.includedUnits]);
+
```

Comment:
liker denne ganske dårlig, hvor vi må ha en ekstra useEffect for å populere verdien fra useQuery. Alternativet jeg ser er å lage enda en komponent som bare håndterer loading state, men liker det enda dårligere da :(